docs/qcfractal/source/glossary.rst

```diff
@@ -45,3 +45,52 @@ Glossary
     DB Table
       A set of data inside the Database which has a common :term:`ObjectId`. The ``table``
       name follows SQL conventions which is also known as a ``collection`` in MongoDB.
+
+    Manager
```

Comment:
Did we need to move these here? If we can have a glossary in multiple locations the previous area might be best to group them in a single location. If they need to stay here, please put them in alphabetical order.